apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.js

```diff
@@ -479,6 +498,186 @@ const RegisterMultiPaymentService = new GQLCustomSchema('RegisterMultiPaymentSer
                 }
             },
         },
+        {
+            access: access.canRegisterMultiPayment,
+            schema: 'registerMultiPaymentForOneReceipt(data: RegisterMultiPaymentForOneReceiptInput!): RegisterMultiPaymentOutput',
+            resolver: async (parent, args, context) => {
+                // wrap validator function to the current call context
+                const throwIf = ({ when, error, extraArgsSupplier }) => throwMutationSpecificErrorIf(
+                    when,
+                    error,
+                    'registerMultiPaymentForOneReceipt',
+                    context,
+                    extraArgsSupplier,
+                )
+                const { data } = args
+                const {
+                    dv,
+                    sender,
+                    receipt,
+                    acquiringIntegrationContext,
+                } = data
+
+                // Stage 0. Check if input is valid
+                throwIf({
+                    when: dv !== 1, error: errors.DV_VERSION_MISMATCH,
+                })
+
+                const senderErrors = validate(sender, SENDER_FIELD_CONSTRAINTS)
+                throwIf({
+                    when: senderErrors && Object.keys(senderErrors).length,
+                    error: errors.WRONG_SENDER_FORMAT,
+                    extraArgsSupplier: () => {
+                        const details = Object.keys(senderErrors).map(field => {
+                            return `${field}: [${senderErrors[field].map(error => `'${error}'`).join(', ')}]`
+                        }).join(', ')
+                        return { messageInterpolation: { details } }
+                    },
+                })
+
+                // Stage 1: get acquiring context & integration
+                const [acquiringContext] = await find('AcquiringIntegrationContext', {
+                    id: acquiringIntegrationContext,
+                })
+
+                throwIf({
+                    when: acquiringContext.deletedAt,
+                    error: errors.ACQUIRING_INTEGRATION_CONTEXT_IS_DELETED,
+                })
+
+                const [acquiringIntegration] = await AcquiringIntegration.getAll(context, {
+                    id: acquiringContext.integration,
+                })
+
+                throwIf({
```

Comment:
Really, you can not get all for deleted integration

Reply:
done